requirements-dev.txt

```diff
@@ -1,4 +1,3 @@
--r requirements.txt
```

Comment:
worth adding a comment in the readme then that for development, both are required to be installed